lib/hammer_cli_foreman/compute_resource.rb

```diff
@@ -31,20 +31,19 @@ class InfoCommand < HammerCLIForeman::InfoCommand
 
       PROVIDER_SPECIFIC_FIELDS = {
         'ovirt' => [
-          Fields::Field.new(:label => _('UUID'), :path => ["compute_resource", "uuid"])
+          Fields::Field.new(:label => _('UUID'), :path => [:uuid])
         ],
         'ec2' => [
-          Fields::Field.new(:label => _('Region'), :path => ["compute_resource", "region"])
+          Fields::Field.new(:label => _('Region'), :path => [:region])
         ],
         'vmware' => [
-          Fields::Field.new(:label => _('UUID'), :path => ["compute_resource", "uuid"]),
-          Fields::Field.new(:label => _('Server'), :path => ["compute_resource", "server"])
+          Fields::Field.new(:label => _('UUID'), :path => [:datacenter])
```

Comment:
> Api doesn't return attribute called uuid here is sample api response(few field truncated)

and the reason is, that it's missing in the API view, that's why I suggested to add `:uuid` to ovirt partial in the other comment

> But after passing uuid='datacenter' allows to create vmware compute resource and works.

I think you can enter the same identifier into either of --datacenter or --uuid, I think it's similar to --domain and --domain-id, but we don't print domain name in domain id field. If you want to display datacenter I'd suggest adding it as separate field and keeping UUID displaying UUID.